kubetest/kops.go

```diff
@@ -462,12 +462,12 @@ func (k kops) Up() error {
 	if len(featureFlags) != 0 {
 		os.Setenv("KOPS_FEATURE_FLAGS", strings.Join(featureFlags, ","))
 	}
+
+	createArgs = append(createArgs, "--yes")
+
 	if err := control.FinishRunning(exec.Command(k.path, createArgs...)); err != nil {
 		return fmt.Errorf("kops create cluster failed: %v", err)
 	}
-	if err := control.FinishRunning(exec.Command(k.path, "update", "cluster", k.cluster, "--yes")); err != nil {
-		return fmt.Errorf("kops update cluster failed: %v", err)
-	}
 
```

Comment:
```suggestion

	// TODO: Once this gets support for N checks in a row, it can replace the above node readiness check
	if err := control.FinishRunning(exec.Command(k.path, "validate", "cluster", k.cluster, "--wait", "5m")); err != nil {
		return fmt.Errorf("kops validate cluster failed: %v", err)
	}

```

Reply:
...and remove from the end.
This way we get to run the better Kops validation first and see also more failure reasons.